cloudcix_primitives/storage_lxd.py

```diff
@@ -1,31 +1,84 @@
 """
-Primitive for Storage drives on LXD hosts
+POC for updating the root disk size of an LXD container.
 """
 # stdlib
 from typing import Tuple
-# lib
+# libs
+from cloudcix.rcc import API_SUCCESS, CHANNEL_SUCCESS, comms_lxd
 # local
-
+from cloudcix_primitives.utils import HostErrorFormatter, LXDCommsWrapper
 
 __all__ = [
-    'build',
-    'read',
-    'scrub',
     'update',
 ]
 
+SUPPORTED_INSTANCES = ['containers']
+
+def update(
+        endpoint_url: str,
+        project: str,
+        instance_name: str,
+        new_size: str,
+        containers: str,
+        verify_lxd_certs: bool = True,
+) -> Tuple[bool, str, dict]:
+    """ Update the root disk size of an LXD container.
+    :param endpoint_url: The endpoint URL for the LXD Host.
+    :param project: The LXD project name.
+    :param instance_name: The name of the LXD instance.
+    :param new_size: The new size for the root disk (e.g., '20GB').
+    :param containers: The type of the LXD instance, currently only 'containers' is supported.
```

Comment:
This does not need to be a paramater, my plan is to remove the param `instance_type` from all LXD primitives and only use `containers` hardcoded into the payloads  e.g.

`rcc.run(cli=f'{instance_type}.get', name=instance_name)` would be updated to `rcc.run(cli=f'containers.get', name=instance_name)`